odoo/cli/db.py

```diff
@@ -0,0 +1,152 @@
+        load.set_defaults(func=self.load)
+        load.add_argument(
+            'database', nargs='?',
+            help="database to create, defaults to dump file's name "
+                 "(without extension)"
+        )
+        load.add_argument('dump_file', help="zip or pg_dump file to load")
+
+        dump = subs.add_parser(
+            "dump", help="Create a dump with filestore.",
+            description="Creates a dump file. The dump is always in zip format "
+                        "(with filestore), to get a no-filestore format use "
+                        "pg_dump directly.")
+        dump.set_defaults(func=self.dump)
+        dump.add_argument('database', help="database to dump")
+        dump.add_argument(
+            'dump_path', nargs='?', default='-',
+            help="if provided, database is dumped to specified path, otherwise "
+                 "or if `-`, dumped to stdout",
+        )
+
+        duplicate = subs.add_parser("duplicate", help="Duplicate a database including filestore.")
+        duplicate.set_defaults(func=self.duplicate)
+        duplicate.add_argument(
+            '-f', '--force', action='store_const', default=False, const=True,
+            help="delete `target` database before copying if it exists"
+        )
+        duplicate.add_argument("source")
+        duplicate.add_argument("target", help="database to copy `source` to, must not exist unless `-f` is specified in which case it will be dropped first")
+
+        rename = subs.add_parser("rename", help="Rename a database including filestore.")
+        rename.set_defaults(func=self.rename)
+        rename.add_argument(
+            '-f', '--force', action='store_const', default=False, const=True,
+            help="delete `target` database before renaming if it exists"
+        )
+        rename.add_argument('source')
+        rename.add_argument("target", help="database to rename `source` to, must not exist unless `-f` is specified, in which case it will be dropped first")
+
+        drop = subs.add_parser("drop", help="Delete a database including filestore")
+        drop.set_defaults(func=self.drop)
+        drop.add_argument("database", help="database to delete")
+
+        args = parser.parse_args(cmdargs)
+
+        config.parse_config([
+            val
+            for k, v in vars(args).items()
+            if v is not None
+            if k in ['config', 'data_dir', 'addons_path'] or k.startswith(('db_', 'pg_'))
+            for val in [
+                '--data-dir' if k == 'data_dir'\
+                    else '--addons-path' if k == 'addons_path'\
+                    else f'--{k}',
+                v,
+            ]
+        ])
+        # force db management active to bypass check when only a
+        # `check_db_management_enabled` version is available.
+        config['list_db'] = True
+        report_configuration()
+
+        args.func(args)
+
+    def load(self, args):
```

Comment:
would be great to add a _check_target here (as well as the command line argument)